README.md

```diff
@@ -2,13 +2,13 @@
 
 This library is an implementation of **boolean-valued predicate functions** in Rust.
 
-[![Build Status](https://travis-ci.org/nastevens/predicates-rs.svg?branch=master)](https://travis-ci.org/nastevens/predicates-rs)
-[![Build status](https://ci.appveyor.com/api/projects/status/yl5w3ittk4kggfsh?svg=true)](https://ci.appveyor.com/project/nastevens/predicates-rs)
+[![Build Status](https://travis-ci.org/assert-rs/predicates-rs.svg?branch=master)](https://travis-ci.org/assert-rs/predicates-rs)
+[![Build status](https://ci.appveyor.com/api/projects/status/yl5w3ittk4kggfsh?svg=true)](https://ci.appveyor.com/project/assert-rs/predicates-rs)
```

Comment:
Should the references to predicate-rs be instead under `epage`?

appveyor isn't as nice as travis in organizing things by org but instead the person who signed up.

Reply:
Interesting, I didn't know there was that restriction. I added through my account too, and it looks like the build is running successfully. I could just leave it as `nastevens`. Thoughts?

Reply:
I grabbed the new key for appveyor pointed at assert-rs, and pushed the change. Let's see how that goes.

Reply:
Sure.

Reply:
Nope, it needs to point at at your build since that's the one being triggered for CI. I'll need your unique URL though, then I can replace to point at `epage`

Reply:
The key is enru6k55xme867u6

Reply:
Looks like we're in business - appveyor kicked off successfully and the badge is pointing to the right spot and seems to be getting the right status 👍